source/manual/alerts/content-performance-manager-app-healthcheck-not-ok.html.md

```diff
@@ -38,5 +38,16 @@ To fix this problem run the [following rake task][2]:
 rake etl:repopulateviews[2018-01-01 2018-01-01]
 ```
 
+## ETL :: no feedex for yesterday
+
+This means the [the ETL master process][1] that runs daily has failed to collect `feedex` metrics from Google Analytics. The issue may originate from the ELT processor responsible for collecting Feedex comments.
```

Comment:
Typo

Reply:
Fixed in https://github.com/alphagov/govuk-developer-docs/pull/1524/commits/31614d89e076013b72737ade38fbfdf4687e5fc3